cli/src/transport.ts

```diff
@@ -51,10 +56,25 @@ function createSerial() {
 
 function createWebSocket(url: string, protocol: string) {
     setupWebsocket()
-    return createWebSocketTransport(url, {
+    const transport = createWebSocketTransport(url, {
         protocols: protocol,
         WebSocket: WebSocket,
     })
+    transport.on(SIDE_DATA, (msg: SideDeviceMessage) => {
+        const { type } = msg
```

Comment:
@mmoskal does the simulator also use this codepath to create a websocket?

Reply:
no, the web socket code for VM is in wasmpre.ts